internal/dataplane/kongstate/plugin.go

```diff
@@ -253,12 +253,12 @@ func RawConfigurationWithNamespacedPatchesToConfiguration(
 			return kong.Configuration{}, err
 		}
 	}
-	return rawConfigToConfiguration(raw)
+	return RawConfigToConfiguration(raw)
 }
 
-// rawConfigToConfiguration decodes raw JSON to the format of Kong configuration.
+// RawConfigToConfiguration decodes raw JSON to the format of Kong configuration.
 // it is run after all patches applied to the initial config.
-func rawConfigToConfiguration(raw []byte) (kong.Configuration, error) {
+func RawConfigToConfiguration(raw []byte) (kong.Configuration, error) {
```

Comment:
Should this move to a generic utility function now that it's no longer just used for plugins?

Reply:
Moved it to `utils.go`.